CONTRIBUTING.md

```diff
@@ -221,6 +221,8 @@ E2E are most useful for testing what happens on a page after it has been loaded
 
 Translations help make Starlight accessible to more people.
 
+Check out the dedicated [i18n contribution guide](https://contribute.docs.astro.build/guides/i18n/#quality-standards--adaptation) for more details regarding our translation process and quality standards.
```

Comment:
@lorenzolewis suggested mentioning that these are the Astro guidelines here too, which feels like a good idea.

(Because I am a lemon, I then proceeded to accidentally delete that comment…)

Reply:
Tweaked the wording to mention the guidelines are from the Astro docs contributor guide like the sentence below (altho not entirely sure this was the suggestion 😅)

Reply:
Yup, that's it. Now it has a lot of "guide" in that one sentence but not a dealbreaker.